tests/infra/AutoCompleteServiceImpl.ts

```diff
@@ -119,4 +118,102 @@ describe('AutocompleteService', () => {
       expect(mockHttpService.get).toHaveBeenCalledWith(expectedFilterUrl, expectedQueryParams);
     });
   });
+});
+
+describe('additionalQueryParams are passed through', () => {
+  const config: AnswersConfig = {
+    apiKey: 'testApiKey',
+    experienceKey: 'testExperienceKey',
+    locale: 'en',
+    additionalQueryParams: {
+      jsLibVersion: 'LIB_VERSION'
+    }
+  };
+  const mockHttpService = new HttpServiceMock();
+  const apiResponseValidator = new ApiResponseValidator();
+
+  it('Universal Autocomplete', async () => {
+    mockHttpService.get.mockResolvedValue(mockAutocompleteResponse);
+    const request: UniversalAutocompleteRequest = {
+      input: ''
+    };
+    const expectedQueryParams = {
```

Comment:
What are your thoughts on using a jest expect.objectContaining for these tests which just checks for the jsLibVersion?

Reply:
yeah I like that more! no point to be redundant